packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -6,7 +6,7 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 import "../interfaces/OptimisticAsserterCallbackRecipientInterface.sol";
 import "../interfaces/OptimisticAsserterInterface.sol";
-import "../interfaces/SovereignSecurityInterface.sol";
```

Comment:
In the comments we still have references to sovereign security. Should those be part of separate PR when adding NatSpec?